README.md

```diff
@@ -118,3 +135,15 @@ F0511 11:47:28.966114       1 server.go:495] open /proc/sys/net/netfilter/nf_con
 
 Make sure to:
 `sudo sysctl net/netfilter/nf_conntrack_max=<value>`, and add a drop-in file to `/etc/sysctl.d/99-custom.conf` to set the kernel parameters permanently.
+
+## Monitoring and metrics
+
+The AddonOperator is instrumented with the prometheus-client provided by controller-runtime to record some useful Addon metrics.
+
+| Metric name                                 | Type       | Description                                                                             |
+|---------------------------------------------|------------|-----------------------------------------------------------------------------------------|
+| `addon_operator_addons_count`               | `GaugeVec` | Total number of Addon installations, grouped by 'available', 'paused' and 'total'       |
```

Comment:
```suggestion
| `addon_operator_addons_count`               | `Gauge` | Total number of Addon installations, grouped by 'available', 'paused' and 'total'       |
```
nit

Reply:
It was intentional since the addon count is a `GaugeVec` (group of gauges partitioned by labels) - https://github.com/openshift/addon-operator/blob/main/internal/metrics/recorder.go#L32

Reply:
Got it. I thought you meant to denote the actual Prometheus metric types (instead of Go types) (https://prometheus.io/docs/concepts/metric_types/#gauge) and `GaugeVec` looked like a typo w.r.t that.